markdown/usage/troubleshooting.md

```diff
@@ -36,9 +37,44 @@ Caused by:
     INFO:    Creating SIF file...
     FATAL:   Unable to pull docker://nfcore/rnaseq:1.3: While searching for mksquashfs: exec: "mksquashfs": executable file not found in $PATH
 ```
-In such a case, please install `mksquashfs` or ask in your IT department that this package is installed for you. 
+
+In such a case, please install `mksquashfs` or ask in your IT department that this package is installed for you.
+
+## Cannot find input files when using Singularity
+
+Depending on how you install Singularity on your system, there are several often occuring caveats. Typically these result in error messages such as these:
+
+```bash
+
+Command error:
+  �[91mERROR  : Failed to resolve path to /home/bla/singularity/mnt/container: No such file or directory
+  �[0m�[31mABORT  : Retval = 255
+  �[0m
+```
+
+You can check to verify that this is indeed a problem by using a Singularity Shell into your container and checking whether your required paths are available **inside** your container:
+
+```bash
+singularity shell /path/to/work/singularity/imagename.simg
+ls -la /path/to/work
+ls -la /path/to/network_storage
+```
+
+If any of these `ls -la` returns a `Not Found` error, you will need to get Singularity Bind Paths set/updated on your system.
+
+The Singularity installation requires certain host paths to be bound. Check out [Singularity Bind Paths Documentation](https://sylabs.io/guides/3.0/user-guide/bind_paths_and_mounts.html) for more information on this in detail. In many cases this can be resolved by adding these paths to your `/etc/singularity/singularity.conf` as illustrated on the document linked above:
```

Comment:
```suggestion
The Singularity installation requires certain host paths to be bound. Please see [Singularity Bind Paths Documentation](https://sylabs.io/guides/3.0/user-guide/bind_paths_and_mounts.html) for a more detailed explanation. In many cases this can be resolved by adding these paths to your `/etc/singularity/singularity.conf` as highlighted in the documentation:
```